ceresdb-protocol/pom.xml

```diff
@@ -27,8 +27,17 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.avro</groupId>
-            <artifactId>avro</artifactId>
+            <groupId>com.github.luben</groupId>
+            <artifactId>zstd-jni</artifactId>
```

Comment:
This seems to be a personal project. Is it reliable enough? Can we remove this dependency?

Reply:
no, zstd lib is required, `zstd-jni` has 633 starts, and itt is recommended in java zstd lib